lib/components/user/nav-login-button.js

```diff
@@ -35,6 +38,10 @@ export default class NavLoginButton extends Component {
     profile: null
   }
 
+  _handleLinkClick = item => {
```

Comment:
Ah, didn't realize this was a handler for an item in a loop. This refactor still creates a function for every item on each render. The way to avoid this for items in a loop is to memorize this handler like this: https://github.com/ibi-group/datatools-ui/blob/48ec0da2e303a9b20a81f3fb757a61d3eef9fa32/lib/manager/components/CreateFeedSource.js#L86

Reply:
Thanks for the suggestion!